pages/osx/defaults.md

```diff
@@ -0,0 +1,23 @@
+# defaults
+
+> Access OS X user defaults
+
+- read system default value
+
+`defaults read {{domain}} {{key}}`
+
+- read default values of applications
```

Comment:
I think this is just a language subtlety. The description is written in a generic sense, not literally referring to a concrete set of multiple applications. That said, it certainly doesn't hurt to be explicit and change the phrasing to refer to a single app, which is what the command actually does.